locale/en/messages.json

```diff
@@ -1019,6 +1022,7 @@
   "{0} must be bigger than or equal to 0": "",
   "{0} of {1}": "",
   "{0} of {1} Cores": "",
+  "{0} seconds": "",
```

Comment:
Why do we have once this and once https://github.com/dcos/dcos-ui/pull/4140/files#diff-3efbfb8074a9a05f9fbe43a2ce435a50R1034 ?

Reply:
I believe because in once case we use named variable and in another case an expression? https://github.com/dcos/dcos-ui/pull/4140/files#diff-9f9e1d6769448e9c866a1ab4b7b664ebR139 @Poltergeist we might want to converge to one version